include/ceed.h

```diff
@@ -148,6 +148,11 @@ typedef enum {
   CEED_MEM_DEVICE,
 } CeedMemType;
 
+static const char *const CeedMemTypes[] = {
```

Comment:
Yeah, make this `extern` and just a declaration, then define it in a file.

Reply:
Oh. Duh.

Thank you